### PR TITLE
Specify segment consumer

### PR DESCRIPTION
**Pull request by MarioPerezEsteso**

### CODE REVIEW

File: src/SegmentServiceProvider.php [hunk 1]
```diff
@@ -35,7 +35,12 @@ public function boot()
         $this->setupConfig();
 
         if ($writeKey = $this->app->config->get('segment.write_key')) {
-            Segment::init($writeKey);
+            $options = [
+                'consumer' => $this->app->config->get('segment.consumer') ?? null,
```
Comment (GrahamCampbell) [inline src/SegmentServiceProvider.php]: No need for the `?? null` I think.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/SegmentServiceProvider.php (hunk 1 above)
Comment (GrahamCampbell) [reply src/SegmentServiceProvider.php]: What if you were to just do an array_only on `$this->app->config->get('segment')`, and then a filter. That'd give you the correct options?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by MarioPerezEsteso** [comment]
Regarding this comment https://github.com/AltThree/Segment/pull/9#discussion_r169700741, what if the config/segment.php had two items? `write_key` and `options` to initialize Segment?

**Comment by GrahamCampbell** [comment]
Thanks. Merged. :)

**Pull request merged**